src/rdkafka_ssl.c

```diff
@@ -999,6 +1000,56 @@ static int rd_kafka_ssl_set_certs (rd_kafka_t *rk, SSL_CTX *ctx,
                 check_pkey = rd_true;
         }
 
+#if OPENSSL_VERSION_NUMBER >= 0x10100000
+        if (rk->rk_conf.ssl.ptr_engine) {
+                STACK_OF(X509_NAME)* cert_names = sk_X509_NAME_new_null();
```

Comment:
Coding style is `type *var`

Reply:
sure, i will make the change in next iteration